src/eradiate/scenes/integrators/_core.py

```diff
@@ -64,3 +64,29 @@ class Integrator(NodeSceneElement, ABC):
         init_type="bool, optional",
         default="False",
     )
+
+    stokes: bool = documented(
+        attrs.field(
+            default=False,
+            converter=attrs.converters.optional(bool),
+        ),
+        doc="If true, calculates and outputs the polarized components of"
+        "the stokes vector. Note that this will only work with a polarized"
+        "mode and a sensor that measures radiance.",
```

Comment:
```suggestion
        doc="If ``True``, calculate and output the polarized components of "
        "the Stokes vector. Note that this will only work with a polarized "
        "mode and a sensor that measures radiance.",
```